rules/windows/privilege_escalation_printspooler_service_suspicious_file.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects exploitation behavior of 3 Privilege Escalation vulnerabilities related to the Print Spooler service. For more
+information lookup the related VulnIDs CVE-2020-1048, CVE-2020-1337 and CVE-2020-1300 and verify that the impacted
+system is patched.
```

```suggestion
Detects attempts to exploit privilege escalation vulnerabilities related to the Print Spooler service. For more
information refer to the following CVE's - CVE-2020-1048, CVE-2020-1337 and CVE-2020-1300 and verify that the impacted
system is patched.
```

This will require a re-lint

You can also add links to the CVEs under `references`

---

Maybe add something on creating files in the respective directories to differentiate this from the other rule